src/elements/common/collaborators/__tests__/withCollaborators.test.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import withCollaborators from '../withCollaborators';
+
+jest.mock('lodash/debounce', () => jest.fn(i => i));
+
+describe('elements/common/withCollaborators', () => {
+    const TestComponent = props => <div {...props} />;
+    const WrappedComponent = withCollaborators(TestComponent);
+    const fileCollaboratorsAPI = {
+        getFileCollaborators: jest.fn(),
+    };
+    const api = {
+        getFileCollaboratorsAPI: () => fileCollaboratorsAPI,
+    };
+    const defaultProps = { api };
+    const getWrapper = props => shallow(<WrappedComponent {...defaultProps} {...props} />);
+    let fileId;
+    let wrapper;
+    let instance;
+    let successCb;
+    let errorCb;
+    beforeEach(() => {
```

Comment:
Can we add some whitespace between logical blocks, similar to our other test files?